pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -59,30 +62,36 @@ let
     owner = "pytorch";
     repo = "cpuinfo";
     # There are no tags in the repository
-    rev = "5916273f79a21551890fd3d56fc5375a78d1598d";
-    hash = "sha256-nXBnloVTuB+AVX59VDU/Wc+Dsx94o92YQuHp3jowx2A=";
+    rev = "3c8b1533ac03dd6531ab6e7b9245d488f13a82a5";
+    hash = "sha256-eshoHmGiu5k0XE/A1SWf7OvBj7/YD9JNSZgoyGzGcLA=";
   };
 
-  flatbuffers = fetchFromGitHub {
-    owner = "google";
-    repo = "flatbuffers";
-    rev = "v1.12.0";
-    hash = "sha256-L1B5Y/c897Jg9fGwT2J3+vaXsZ+lfXnskp8Gto1p/Tg=";
+  pytorch_clog = effectiveStdenv.mkDerivation {
+    pname = "clog";
+    version = "3c8b153";
+    src = "${pytorch_cpuinfo}/deps/clog";
+
+    nativeBuildInputs = [ cmake gbenchmark gtest ];
+    cmakeFlags = [
+      "-DUSE_SYSTEM_GOOGLEBENCHMARK=ON"
+      "-DUSE_SYSTEM_GOOGLETEST=ON"
+      "-DUSE_SYSTEM_LIBS=ON"
+    ];
```

Comment:
probably cleaner to factor out `pytorch_cpuinfo` rev & share that between the source & the `clog` version string.

also even though `pytorch_clog` doesn't seem to use `gbenchmark` it'll fail without `-DUSE_SYSTEM_GOOGLEBENCHMARK=ON` so we have to leave it there